websites/A/Animeflix/presence.ts

```diff
@@ -32,21 +32,11 @@ presence.on("UpdateData", async () => {
 		}
 		default:
 			if (pathname.includes("/watch")) {
-				const episodeName = document.querySelector<HTMLMetaElement>(
-					'meta[name="anime-skip.episode.name"]'
-				);
-				presenceData.details = `Watching ${
-					document.querySelector<HTMLMetaElement>(
-						'meta[name="anime-skip.show.name"]'
-					).content
-				}`;
-				presenceData.state = episodeName.content.includes("Episode")
-					? episodeName.content
-					: `${
-							document.querySelector<HTMLMetaElement>(
-								'meta[name="anime-skip.episode.number"]'
-							).content
-					  }. ${episodeName.content}`;
+				presenceData.details = `Watching ${JSON.parse(
+					document.querySelector<HTMLMetaElement>('script[id="syncData"]')
+						.textContent
+				).name.replace(/\b[a-z]/g, (letter: string) => letter.toUpperCase())}`;
+				presenceData.state = "Watch for free on Animeflix!";
```

Comment:
Actually remove this, this is seen as advertising

Reply:
Do you mean the "Watch for free on Animeflix!" part?

Reply:
Yes

Reply:
Okay, I removed it